lib/src/components/NxBinaryDonutChart/NxBinaryDonutChart.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { forwardRef } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxBinaryDonutChart.scss';
+
+const NxBinaryDonutChart = forwardRef<SVGSVGElement, Props>(
+    function NxBinaryDonutChart(props, ref) {
+      const { innerRadiusPercent, percent, ...svgAttrs } = props;
+      const innerRadiusPercentNotNull = innerRadiusPercent == null ? 50 : innerRadiusPercent < 0 ? 0 :
+        innerRadiusPercent > 100 ? 100 : innerRadiusPercent;
+
+      const strokeWidth = 100 - innerRadiusPercentNotNull;
+      const r = innerRadiusPercentNotNull + (strokeWidth / 2);
+
+      // Add 0.5π because we want the angle from the top of the circle, not the right
+      const arcEndAngle = (Math.PI / 2) + (2 * Math.PI * (percent / 100));
+      const arcEndX = r * Math.cos(arcEndAngle);
+      // Multiply by -1 because the y-axis origin (0,0) is at the top-left rather than bottom-left
```

Comment:
This comment isn't quite right, the origin is in the center using this viewbox.  I'd say y increases going downwards on the screen, not upwards

Reply:
> This comment isn't quite right, the origin is in the center using this viewbox. I'd say y increases going downwards on the screen, not upwards

Changed 0233f50e78105bae19a8d84cce928a4d31c13944